lib/src/test/java/io/ably/lib/transport/HostsTest.java

```diff
@@ -65,7 +66,7 @@ public void hosts_fallback_null_array(){
 
     @Test
     public void hosts_fallback_empty_array(){
-        String host = Hosts.getFallback(null, new ArrayList<>());
+        String host = Hosts.getFallback(null, Collections.<String>emptyList());
```

Comment:
This code was written in the previous commit so should be squished unless this introduces some new concept I believe.